src/frontend/src/views/Privacy/index.tsx

```diff
@@ -31,17 +31,17 @@ const PrivacyPolicy = (): JSX.Element => {
         <H4>Last Updated: May 17, 2021. </H4>
       </Title>
       <P>
-        The Chan Zuckerberg Initiative Foundation, a 501(c)(3) nonprofit private
-        foundation (&quot;CZIF,&quot; &quot;we,&quot; &quot;us,&quot; or
-        &quot;our&quot;), provides the Aspen product (&quot;Services&quot; or
-        &quot;Aspen&quot;) in close collaboration with the Chan Zuckerberg
-        Biohub (&quot;CZB&quot;), and the Chan Zuckerberg Initiative, LLC
-        (&quot;CZI LLC&quot;). This Privacy Policy describes the types of
-        information we collect or that is uploaded by Aspen Users (collectively
-        &quot;Users&quot; or &quot;you&quot;, ex: registered public health
-        officials at state and/or county level Departments of Public Health
-        (&quot;DPH&quot;), other public health researchers), and how we use,
-        share, and protect that information.
+        The Chan Zuckerberg Initiative private foundation (&quot;<B>CZIF</B>,
```

Comment:
In the AgreeTerms, you're using `&rdquo;` and `&ldquo;`, while everywhere else seems to be using `&quot;`. Is there a reason why it changes? For this kind of copy, I'd assume they should just all the same.

And if you're using plain old `&quot;`, can we just use `"` symbols instead? Maybe the linter will scream its head off, but I certainly find that more readable if it's not going to screw up some escaping somewhere.